.travis.yml

```diff
@@ -1,10 +1,28 @@
+sudo: required
+
 services:
   - docker
 
 language: python
 python:
   - "3.5"
 
+before_install:
+- docker pull diblab/eel-pond:0.1
```

Comment:
You can deposit the binary image produced by `docker build <blah>` on docker hub and anyone can then fetch it with `docker pull ...`. The dockerhub entry doesn't have to be linked to any github repo or similar so that is why it looks so empty over there. (You could link the dockerhub entry to a repo and then [dockerhub builds a new image for every new commit](https://docs.docker.com/docker-hub/builds/) to `master` in that repo). In that case you end up seeing the `Dockerfile` and stuff also on the dockerhub. Does that make sense?